packages/one-app-hmr/lib/webpack/scripts/hot-client.js

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2020 American Express Travel Related Services Company, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { subscribe } from 'webpack-hot-middleware/client';
+
+function formatPayloadMessage(payload) {
+  return ['[One App HMR]:', payload.action, payload.path].join(' ');
+}
+
+subscribe((payload) => {
+  switch (payload.action) {
+    // TODO: update store state of updated lang pack, or reload is required
+    case 'locale:add':
+    case 'locale:change':
+    case 'locale:remove':
+    case 'parrot:add':
+    case 'parrot:change':
+    case 'parrot:remove':
+      console.log(['%c', formatPayloadMessage(payload)].join(' '), 'background: #222; color: #bada55');
+      break;
+    default:
+      break;
+  }
+});
```

Comment:
This is a script ran on the client, it is how we communicate via [server sent events](https://developer.mozilla.org/en-US/docs/Web/API/Server-sent_events/Using_server-sent_events). It is how we would tell the client of updates to either locales or scenarios and update. subsequently.